base/src/main/java/com/smartdevicelink/protocol/enums/ControlFrameTags.java

```diff
@@ -44,6 +44,7 @@
 	private static class NAKBase{
 		/** An array of rejected parameters related to the corresponding request*/
 		public static final String REJECTED_PARAMS = "rejectedParams";
+		public static final String REASON = "reason";
```

Comment:
`RegisterSecondaryTransportNAK` already has a `reason` so it should be removed from that specific NAK as it exists in the base class now.